_includes/footer.md

```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```

Comment:
I think we should still have the legalese on the index page as it is expected to be present. We probalby don't need it in every footer. Maybe we can include that "partial" (I forget what Jekyll calls it) in the footer just on the index page?